### DISCUSSION TITLE
「ストックする」を1アクションでできる過去のUXもサポートしてほしい

### DISCUSSION THREAD

**Discussion by vsanna** [increments/qiita-discussions#114] [要望・アイデア・不具合]
## why
後で読み返したい記事としてストックする場合、過去には1クリックでストックできましたが、現在はモーダル上でもう1クリックする必要があります。
ストックをカテゴライズする利便性は理解しつつ、1クリックでデフォルト設定したカテゴリとしてストックする機能を希望します

## what to expect
「ストックする」ボタンを押した際に、予め登録してあるデフォルトカテゴリとしてストックされる。

**Comment by kskwtnk** [comment]
@vsanna 
ご意見ありがとうございます！

ストック機能の改善は絶対に実施するつもりなのですが、今行っている施策がだいぶ重めでして……着手まで時間がかかってしまいそうです 🙏 

ちなみに、現状ストックにおいて実施したいと思っているのが

- ストックまでのクリック回数を減らす
- カテゴリーの並び変え

こちらとなっています！

**Comment by vsanna** [comment]
@xrxoxcxox 
ありがとうございます 🙏 
後で読むための手数が億劫で、(良くないとおもいつつ）lgtmかわからない未読の記事をlgtmする習慣が個人的についてしまっているのもあり、対応事項として検討していただけるとありがたい限りです。

**Comment by kyntk** [comment]
@vsanna 
大変お待たせいたしました！
ストックが1アクションでできるようになりました :sparkles: 
ぜひお試しください！
https://twitter.com/Qiita/status/1666410919025586178

ストックされるカテゴリーとしては、まずはデフォルトで「あとで読む」にストックされるようになっています。
今後の改善として、ユーザーごとにデフォルトカテゴリーを選択できるようにすることも検討しています！
